### PR TITLE
Save button using Square-Tape 

### LINKED ISSUE #15: Implement an in-app saving and export function for badges
Currently badges cannot be saved. Please add a save button. It would make it easier if the user does not necessarily has to add a name. However the name of the saved file  - it should be possible to change it. Also ensure the user can export the saved file and open it e.g. on another phone.

### PR DESCRIPTION
**Pull request by akkywadhwa**
Fixes #15 

Changes:
- Save Button Added
- Saves the file in ../../../com.nilhcem.blenamebadge/files

Screenshots for the change:
Screenshot_20190401-145254_Badge Magic

### CODE REVIEW

File: app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt [hunk 1]
```diff
@@ -101,6 +107,43 @@ class MessageActivity : AppCompatActivity() {
...
+                else if(marquee.isChecked)
+                    qFile.add("Type - Marquee".toByteArray())
+                else
+                    qFile.add("Type - None".toByteArray())
+
+                val sb = StringBuilder()
+                sb.append("Records saved successfully with file name ")
+                sb.append(fileName)
+
+                Toast.makeText(this,sb.toString(),Toast.LENGTH_SHORT).show()
```
Comment (iamareebjamal) [inline app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: Use POJO and JSON for saving items, and don't create a file for each item

**Comment by adityastic** [comment]
@yatri1609 has opened it already I guess. #92

**Comment by akkywadhwa** [comment]
@adityastic I used it doing square-tape. She did it with CSV.

**Comment by adityastic** [comment]
@akkywadhwa  Yes Sir, I just suggest asking if she is still working on it or not. The last I did, she told me that she was. Good work though.

**Comment by yatri1609** [comment]
@akkywadhwa I'm working on this

File: app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt (hunk 1 above)
Comment (akkywadhwa) [reply app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt]: @iamareebjamal Will it be okay if I and @yatri1609 work on the same issue and then you can decide which PR can be merged.

**Comment by cweitat** [comment]
any updates?

**Comment by adityastic** [comment]
@iamareebjamal this can be closed now

**Pull request closed without merging**